pages/community/gsoc.js

```diff
@@ -0,0 +1,100 @@
+import GenericLayout from '../../components/layout/GenericLayout'
+import Heading from '../../components/typography/Heading'
+import Paragraph from '../../components/typography/Paragraph'
+
+export default function GSoCPage() {
+
+  return (
+      <div className="py-12 overflow-hidden">
```

Comment:
You need to put this whole content inside `GenericLayout` component to use it. You have imported it but not used it here. Do [check this](https://github.com/asyncapi/website/blob/master/pages/roadmap.js#L28) for further explanation.